### PR TITLE
Update: Schedule no delivery

### PR DESCRIPTION
**Pull request by holderb5**
Resolves #

## Proposed Changes

-

## Screenshots

Screen Shot 2021-12-20 at 12 57 48 PM

Screen Shot 2021-12-20 at 12 56 20 PM

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jkusa** [comment]
@holderb5 can you please post a screenshot?

File: packages/reports/addon/components/common-actions/schedule.hbs [hunk 1]
```diff
@@ -59,21 +59,53 @@
...
+            @triggerClass="schedule__modal-delivery-trigger w-full"
+            @dropdownClass="w-full"
+            @options={{this.deliveryOptions}}
+            @selected={{this.localDeliveryRule.delivery}}
+            @renderInPlace={{true}}
+            @onChange={{fn (mut this.localDeliveryRule.delivery)}}
+            as | delivery |
+          >
+            {{capitalize delivery}}
+          </PowerSelect>
```
Comment (jkusa) [inline packages/reports/addon/components/common-actions/schedule.hbs:74]: Can we add some description text below the selection for the two options.

File: packages/reports/addon/components/common-actions/schedule.hbs [hunk 2]
```diff
@@ -59,21 +59,53 @@
...
         </DenaliInputGroup>
+        {{#if (eq this.localDeliveryRule.delivery 'email')}}
```
Comment (daniellevinson) [inline packages/reports/addon/components/common-actions/schedule.hbs:76]:
```suggestion
        {{#if (eq this.localDeliveryRule.delivery "email")}}
```

File: packages/reports/addon/components/common-actions/schedule.hbs [hunk 3]
```diff
@@ -90,23 +122,7 @@
...
-            @searchEnabled={{false}}
-            @onChange={{this.updateFormat}}
-            @renderInPlace={{true}}
-            as | format |
-          >
-            {{format}}
-          </PowerSelect>
-        </DenaliInputGroup>
-        {{#if (feature-flag "enabledNotifyIfData")}}
+        {{#if (and (feature-flag "enabledNotifyIfData") (eq this.localDeliveryRule.delivery 'email'))}}
```
Comment (daniellevinson) [inline packages/reports/addon/components/common-actions/schedule.hbs:125]:
```suggestion
        {{#if (and (feature-flag "enabledNotifyIfData") (eq this.localDeliveryRule.delivery "email"))}}
```

File: packages/reports/addon/components/common-actions/schedule.hbs [hunk 4]
```diff
@@ -59,21 +59,56 @@
...
+            @options={{this.deliveryOptions}}
+            @selected={{this.localDeliveryRule.delivery}}
+            @renderInPlace={{true}}
+            @onChange={{fn (mut this.localDeliveryRule.delivery)}}
+            as | delivery |
+          >
+            {{capitalize delivery}}
+          </PowerSelect>
+          <div class="schedule__modal-helper-delivery">
+              If 'Email' is selected emails will be automatically delivered to the designated recipients, to not recieve emails select 'None'.
```
Comment (jkusa) [inline packages/reports/addon/components/common-actions/schedule.hbs:76]: Can we show only the description of the option selected?

Comment (jkusa) [reply packages/reports/addon/components/common-actions/schedule.hbs:76]: Email - `Send this ${report/dashboard} as email on a recurring basis`
None - `Schedule and cache this ${report/dashboard} without delivery`

File: packages/reports/addon/components/common-actions/schedule.hbs [hunk 5]
```diff
@@ -59,21 +59,62 @@
...
+              @selected={{this.localDeliveryRule.delivery}}
+              @renderInPlace={{true}}
+              @onChange={{fn (mut this.localDeliveryRule.delivery)}}
+              as | delivery |
+            >
+              {{capitalize delivery}}
+            </PowerSelect>
+          </DenaliInputGroup>
+          <div class="schedule__modal-helper-delivery">
+            {{#if (eq this.localDeliveryRule.delivery "email")}}
```
Comment (jkusa) [inline packages/reports/addon/components/common-actions/schedule.hbs:78]: If this does not fit under the input box, can we stack the input group?

**Review by cythrawll — APPROVED** [review]

**Review by kevinhinterlong — APPROVED** [review]

**Pull request merged**